account_wallet/models/account_move_line.py

```diff
@@ -0,0 +1,108 @@
+# © 2015  Laetitia Gangloff, Acsone SA/NV (http://www.acsone.eu)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.osv.expression import FALSE_DOMAIN
+from odoo.tools import float_compare
+from odoo.tools.translate import _
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    account_wallet_id = fields.Many2one(
+        comodel_name="account.wallet",
+        string="Wallet",
+        ondelete="restrict",
+        index=True,
+    )
```

Comment:
I was thinking about adding a domain on that field.

I mean : 

- you have a wallet A for a customer A and a wallet B for a customer B.
- You make a new invocie for the customer, and want to sale a wallet product again.
- Does it make sense to have the possibility to select the wallet B ? 

Something like : 

```domain = [
('partner_id', '=', False), # anonymous.
('partner_id', '=', partner_id),
]
```

What do you think ?

Reply:
done in #9